### PR TITLE
feat: update terms of use

### PR DESCRIPTION
**Pull request by Nebulis**
Updating terms of use

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ module.exports = withBundleAnalyzer(
         "/collaborate": { page: "/collaborate" },
         "/privacy": { page: "/privacy" },
         "/terms": { page: "/terms" },
+        "/collaborators-terms": { page: "/collaborators-terms" },
```
Comment (sumitnitsurat) [inline next.config.js]: How do I access this link ? Can not any navigation link on the website?

Comment (Nebulis) [reply next.config.js]: haha :p only through terms of use page there is a link in the schedule part that lead to that page (https://github.com/OpenCerts/opencerts-website/pull/425/files#diff-6bf9c6a3053fa5d96f12a3e6a72d6515R490)

I should have added that in the description, my bad

File: pages/collaborators-terms.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    description:
+      "These Terms of Use govern your way of collaborating on opencerts",
+    url: `${URL}/collaborators-terms`
+  }
+};
+
+const CollaboratorsTermsPage = () => (
+  <>
+    <NextSeo config={merge(DEFAULT_SEO, PAGE_SEO)} />
+    <NavigationBar active="collaborate" />
```
Comment (sumitnitsurat) [inline pages/collaborators-terms.js]: Some strange behavior. When I go to /collaborators-terms page, in the header collaborate tabs get highlighted and become non-clickable while the link on the bottom doesn`t.

**Comment by sumitnitsurat** [comment]
just one comment about the UI. rest looks good to me.

File: pages/collaborators-terms.js (hunk 2 above)
Comment (Nebulis) [reply pages/collaborators-terms.js]: I need to remove this thanks 👍

Comment (Nebulis) [reply pages/collaborators-terms.js]: updated

**Pull request merged**